tests/testImageDecorrelation.py

```diff
@@ -190,65 +182,124 @@ def setUp(self):
                                                                      "DETECTED", "BAD",
                                                                      "NO_DATA", "DETECTED_NEGATIVE"]))
 
+    def _setUpImages(self, var1=0.04, var2=0.04):
+        """!Generate a fake aligned template and science image and analyse the noise in the
+        resulting diffim. First, use a non-spatially-varying psf.
+        """
+
+        self.var1 = var1  # variance of noise in science image
+        self.var2 = var2  # variance of noise in template image
+
+        self.im1ex, self.im2ex \
+            = makeFakeImages(var1=self.var1, var2=self.var2, psf1=self.psf1_sigma, psf2=self.psf2_sigma,
+                             n_sources=50, verbose=False)
+
     def tearDown(self):
         del self.im1ex
         del self.im2ex
 
-    def testDiffimCorrection(self):
-        """! Check that the variance of the corrected diffim matches the theoretical value
-        (to within a 1% tolerance).
-        """
-
+    def _testImages(self):
         # Create the matching kernel. We used Gaussian PSFs for im1 and im2, so we can compute the "expected"
```

Comment:
Though it's a "private" method, it would be good to have a short docstring here that says what gets tested.